gatsby-config.js

```diff
@@ -55,10 +55,21 @@ module.exports = {
         start_url: '/',
         // background_color: '#663399',
         // theme_color: '#663399',
-        display: 'minimal-ui',
+        display: 'minimal-ui'
         // icon: 'src/images/gatsby-icon.png', // This path is relative to the root of the site.
-      },
+      }
     },
     'gatsby-plugin-offline',
-  ],
+    {
+      resolve: 'gatsby-plugin-eslint',
```

Comment:
@alexdor  I added `gatsby-plugin-eslint` in order the eslint to run  at the dev environment.
Is this something we wanted to avoid?

Reply:
@alexdor @dPliakos I merged this pr because it had changes in multiple files and would be very difficult to resolve conflicts later (the changes are not all auto generated by the linter). We can change the configuration in another pr if we need to.